src/Concurrency/TasksOneToOneOPENMP.hpp

```diff
@@ -41,6 +41,9 @@ void TasksOneToOne<Threading::OPENMP>::operator()(F&& f, Args&&... args)
       throw std::runtime_error(nesting_error);
   int nested_throw_count = 0;
   int throw_count = 0;
+  // Nor should nested threading be used when a task block is active
+  int outside_nested = omp_get_nested();
```

Comment:
You need to be very careful with this change because it also affects also the first level.
Based on https://www.openmp.org/spec-html/5.0/openmpsu120.html about omp_get_nested,
this routine returns true if max-active-levels-var is greater than 1 for the current task.
If I remember correctly, quite a few widely used OpenMP runtimes default max-active-levels-var to some number like 2^32-1 if OMP_MAX_ACTIVE_LEVELS environment doesn't exist. Namely, what you add here will not take any effect.
If the default of max-active-levels-var is 1, then what you changed will turn off all the threading even the first (crowd) level. In addition omp_get_nested is deprecated in 5.0. Its use must be restricted.